src/mlte/measurement/__init__.py

```diff
@@ -2,10 +2,12 @@
 from .measurement_metadata import MeasurementMetadata
 from .identifier import Identifier
 from .process_measurement import ProcessMeasurement
+from .external_measurement import ExternalMeasurement
 
 __all__ = [
     "Measurement",
     "MeasurementMetadata",
```

Comment:
We should now be able to remove `MeasurementMetadata` and `Identifier` from the `measurement` module-level exports. I think that since we now have your mechanism for integrating external measurements in a more "familiar" way, we should just hide these details from the user entirely. If you are in agreement, let's modify this file to remove the exports.